fmriprep/info.py

```diff
@@ -71,6 +71,7 @@
     'statsmodels',
     'nipype',
     'seaborn',
+    'indexed_gzip',
```

Comment:
`indexed_gzip>=0.6.1`

You can get a slowdown on some operations with 0.6.0 or earlier.